appcues/src/main/java/com/appcues/ui/primitive/ExperiencePrimitive.kt

```diff
@@ -65,12 +67,18 @@ internal fun ExperiencePrimitive.Compose(matchParentBox: BoxScope? = null) {
                 is ImagePrimitive -> Compose(modifier, matchParentBox)
                 is TextPrimitive -> Compose(modifier)
                 is VerticalStackPrimitive -> Compose(modifier)
+                is TextInputPrimitive -> throw NotImplementedError()
+                is OptionSelectPrimitive -> throw NotImplementedError()
             }
         }
     }
 }
 
 private fun ExperiencePrimitive.getRole(): Role {
+    // This is for any tap actions in the experience which currently only
```

Comment:
yes, sorry I had no comments here you probably had to do some searching to figure it out whats that about.